hmy/hmy.go

```diff
@@ -190,6 +194,7 @@ func (hmy *Harmony) GetNodeMetadata() commonRPC.NodeMetadata {
 		Archival:       cfg.GetArchival(),
 		NodeBootTime:   hmy.NodeAPI.GetNodeBootTime(),
 		PeerID:         nodeconfig.GetPeerID(),
+		Consensus:      consensusInternal,
```

Comment:
let's not put this in metadata, reason is in a different comment.

Reply:
I'll remove the IsSync variable then.